docs/src/main/asciidoc/security-authentication-mechanisms.adoc

```diff
@@ -425,13 +434,17 @@ In both cases, `quarkus-oidc` requires a connection to the specified OpenID Conn
 * If the user authentication requires Authorization Code flow, or you need to support multiple tenants, use `quarkus-oidc`.
 `quarkus-oidc` can also request user information by using both Authorization Code Flow and Bearer access tokens.
 
-* If your bearer tokens must be verified, use `quarkus-oidc`, `quarkus-smallrye-jwt`, or `quarkus-elytron-security-oauth2`.
+* If your bearer tokens must be verified, use `quarkus-oidc`,
+ifndef::quarkus-elytron-security-oauth2[`quarkus-elytron-security-oauth2`, ]
+or `quarkus-smallrye-jwt`.
```

Comment:
Here we'll end up with `quarkus-oidc, or quarkus-smallrte-jwt` so if possible, drop `,` somehow